spec/unit/module_spec.rb

```diff
@@ -440,6 +440,50 @@
     end
   end
 
+
+  describe "i18n" do
+
+    let(:modpath) { tmpdir('modpath') }
+    let(:modname) { 'puppetlabs-i18n'}
+
+    before do
+      modroot = "#{modpath}/#{modname}/"
+      @config_path = "#{modroot}/locales/config.yaml"
```

Comment:
nit: These variables should probably all be `let` definitions.  None of them are used in `before(:after)` which is usually why `@` variables are used.